File: php/09_strings.php
<?php
// 09 strings

$name = "ulas";
echo strtoupper($name) . "\n";
echo strtolower($name) . "\n";
echo strlen($name) . "\n";

$age = 20;
echo "$name is $age years old\n";

echo substr($name, 0, 2) . "\n"; // ul
echo strrev($name) . "\n";

echo (str_contains($name, "ula") ? "yes" : "no") . "\n";
echo (str_starts_with($name, "u") ? "yes" : "no") . "\n";

File: cpp/09_strings.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {
    string name = "ulas";

    string upper = name;
    transform(upper.begin(), upper.end(), upper.begin(), ::toupper);
    cout << upper << endl;

    string lower = name;
    transform(lower.begin(), lower.end(), lower.begin(), ::tolower);
    cout << lower << endl;

    cout << name.length() << endl;

    int age = 20;
    cout << name << " is " << age << " years old" << endl;

    cout << name.substr(0, 2) << endl; // ul

    string rev = name;
    reverse(rev.begin(), rev.end());
    cout << rev << endl;

    cout << (name.find("ula") != string::npos ? "yes" : "no") << endl;
    cout << (name.rfind("u", 0) == 0 ? "yes" : "no") << endl;

    return 0;
}
